README.md

```diff
@@ -927,6 +927,115 @@ constructor(props) {
 
 `componentWillUnmount()` - is invoked immediately before a component is unmounted and destroyed. Useful for resource cleanup.
 
+### Vue.js
+
+#### beforeCreate
+
+Called synchronously immediately after the instance has been initialized, before data observation and event/watcher setup.
+
+```javascript
+new Vue({
+  beforeCreate: function () {
+    // `this` points to the vm instance
```

Comment:
No reason to put the same comment in every method, Could you export this info to the first paragraph?

Reply:
i wondered should i just put this info only in first paragraph or not, because may be someone jumped into certain lifecycle without reading the info from start. i put that info in every method so that someone won't miss it. that's my thought.

Reply:
Yeah, that's kind of true, but on the other hand, if someone jumps directly to a specific method I assume he's familiar with the framework to some degree and thus knows what `this` is inside it.

Reply:
ah, that's right. i'll make some changes asap. 🚀